README.md

```diff
@@ -40,364 +43,346 @@ some kind of discriminant:
   - `os.platform()` (`"win32"`, `"linux"`, `"darwin"`, etc.)
   - `"constants"` module (`ENOENT`, `EEXIST`, etc.; `S_IFMT`, `S_IFREG`, etc.)
 
+In addition, with the recent adoption of [TypeScript Type Stripping][] in NodeJS, there has been renewed interest in
```

Comment:
```suggestion
In addition, with the recent adoption of [TypeScript Type Stripping][https://nodejs.org/docs/latest/api/typescript.html#type-stripping] in NodeJS, there has been renewed interest in
```

Optional:  You may also wish to mention that Node's type-stripping was inspired by the techniques in [ts-blank-space](https://bloomberg.github.io/ts-blank-space/) given some of the committee understand the type-stripping concept under that name.

Reply:
This isn't necessary. The `[text][]` syntax in markdown uses the named definitions at the end of the document. `[TypeScript Type Stripping][]` uses the `[TypeScript Type Stripping]: https://nodejs.org/docs/latest/api/typescript.html#type-stripping` definition on line 446